README.md

```diff
@@ -39,7 +50,16 @@ After you make your changes, click **Commit changes**.
 This will automatically create a [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) of the docs in your GitHub account with the changes.
 
 Once your edits are ready in GitHub, follow the prompts to **create a pull request** and submit your changes for review.
-Every pull request needs to be reviewed by our contributors.
+Every pull request needs to be reviewed by our contributors and approved by a Maintainer.
```

Comment:
Is the capitalisation difference intentional here because of governance roles or something? Contributor and maintainer are both defined there right? I’d personally just go with lowercase here, but am not tied to that.

```suggestion
Every pull request needs to be reviewed by our contributors and approved by a maintainer.
```